src/bindist/fdppauto.bat

```diff
@@ -4,6 +4,7 @@ path %DOSEMUDRV%:\dosemu;%SHELLDRV%:\;
 if not "%FREEDOSDRV%" == "" path %PATH%%FREEDOSDRV%:\bin;%FREEDOSDRV%:\gnu;
 if not "%USERDRV%" == "" set TEMP=%USERDRV%:\tmp
 prompt $P$G
+emufs.com
```

Comment:
The standard fdppauto.bat isn't used in the tests. You'll need to modify the one in common_framework.py